### PR TITLE
Show a warning to admins when attempting to complete orders

### LINKED ISSUE #307: Aid package shows $0 pledged, but has moved to the Goal Reached section.
- [ ] Aid package shows $0 pledged, but has moved to the Goal Reached section. 
Screen Shot 2022-07-29 at 17 25 59

Screen Shot 2022-07-29 at 17 25 45

### PR DESCRIPTION
**Pull request by dinuka-rp**
## Purpose
The purpose of this PR is to fix #307 

## Goals
Show a warning to admins when attempting to complete orders

### CODE REVIEW

**Comment by dinuka-rp** [comment]
@nshamit Is it required to check if a percentage of the pledge has been completed when showing this warning message?
Currently, it's shown regardless of the completion percentage of the required total goal.

File: admin-portal/src/pages/packageDetails/packageDetails.tsx [hunk 1]
```diff
@@ -85,6 +85,14 @@ export default function PackageDetails() {
       `Are you sure you want to change the status to ${statusToBeChanged}?`
     );
     if (confirmed) {
+      if (statusToLevel[statusToBeChanged] > 3) {
+        const confirmationCheck2 = window.confirm(
```
Comment (jayasanka-sack) [inline admin-portal/src/pages/packageDetails/packageDetails.tsx:89]: Let's add a meaningful variable name

**Comment by jayasanka-sack** [comment]
@dinuka-rp The requirement is to prevent users from completing a package with 0 pledges. Therefore we need to check if the amount is 0, if yes we should not allow users to change the status.

**Comment by jayasanka-sack** [comment]
Saw @nshamit 's comment later:
https://github.com/LSFLK/MedicinesforLK/issues/307#issuecomment-1199234212

Your implementation seems correct.

**Review by jayasanka-sack — APPROVED** [review]
Thanks @dinuka-rp

**Comment by jayasanka-sack** [comment]
@dinuka-rp The validation message is showing on every completion step. ex: we don't need it when moving from `shipped` to `delivered`

It should display the confirmation only when changing to a completion state from `draft` or `Published`

**Pull request merged**